frontend/src/common.styles.ts

```diff
@@ -0,0 +1,54 @@
+import * as tokens from '@bcgov/design-tokens/js';
+import { css } from '@emotion/react';
+
+export const componentContainer = css({
+  display: 'flex',
+  flexDirection: 'column',
+  marginLeft: '3rem',
+  marginRight: '3rem',
+  maxWidth: '992px',
+});
+
+export const formCss = css({
+  marginTop: '1rem',
+  width: '100%',
+
+  '.bcds-react-aria-TextField': {
+    width: '100%',
+  },
+
+  '.bcds-react-aria-Select': {
+    Button: {
+      width: '100%',
+    },
+  },
+});
+
+export const hideCheckboxGroup = css({
+  height: '0',
+  overflow: 'hidden',
+  width: '0',
+});
+
+export const showCheckboxGroup = css({
+  '> label': {
+    marginLeft: '2rem',
```

Comment:
Most of our margins use px as the unit instead of rem. I understand that rem is more flexible; are you hoping to change everything to use rem as the unit?

Reply:
I was hoping to switch over everything to rem eventually, on a piece-by-piece basis.